pombola/kenya/management/commands/kenya_import_2017_election_results.py

```diff
@@ -0,0 +1,295 @@
+    'party:23': 'ford',
+    'party:21': 'odm',
+    'party:15': 'party-development-and-reform',
+    'party:62': 'amani_national_congress',
+    'party:12': 'wdm-k',
+    'party:36': 'ccu',
+    'party:48': 'economic-freedom-party',
+    'party:67': 'maendeleo-chap-chap-party',
+    'party:39': 'pdp',
+    'party:9': 'knc',
+    'party:54': 'chama-cha-mashinani',
+    'party:31': 'kenya-patriots-party',
+    'party:50': 'muungano-party',
+    'party:40': 'nd',
+    'party:46': 'frontier-alliance-party',
+    'party:37': 'nap-k',  # 'napk' seems to be the same body, but less complete
+    'party:13': 'democratic-party-kenya',
+    'party:14': 'pnu',
+}
+
+PARTY_MEMBER_TITLE = PositionTitle.objects.get(slug='member')
+
+YNR_ID_SCHEME_NAME = 'ynr-ke'
+
+PERSON_CONTENT_TYPE = ContentType.objects.get_for_model(Person)
+
+
+class bcolors:
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
+
+
+class Command(NoArgsCommand):
+    help = 'Import elected representatives from the 2017 election'
+
+    def handle_noargs(self, **options):
+
+        for import_type in import_types:
+
+            csv_path = os.path.join(results_directory, import_type['file'])
+
+            with open(csv_path) as csv_file:
+                for row in csv.DictReader(csv_file):
+
+                    print 'Importing {}'.format(row['name'])
+
+                    # Let's sanity check some important things.
+
+                    # Party?
+
+                    if row['party_id'] != 'party:IND':
+
+                        if row['party_id'] in PARTY_MAP:
+                            party = Organisation.objects.get(slug=PARTY_MAP[row['party_id']])
+                        else:
+                            raise Exception('Party {} ({}) is not in map!'.format(row['party_name'], row['party_id']))
+
+                    # Area?
+
+                    print '\tTrying to find {} "{}"'.format(import_type['area_type'], row['post_label'])
+
+                    area_slug = slugify(row['post_label']) + import_type['area_suffix']
+
+                    try:
+                        area = Place.objects.get(
+                            slug=area_slug,
+                            kind=import_type['area_type']
+                        )
+                    except ObjectDoesNotExist:
+                        print '\tArea not found, creating!'
+                        area = Place(
+                            slug=area_slug,
+                            name=row['post_label'],
+                            kind=import_type['area_type']
+                        )
+
+                    # If the area has a term, set that
+                    if import_type['area_has_term']:
+                        # This is a tuple from get_or_create, hence the [0]
+                        area.parliamentary_session = PARLIAMENTARY_SESSION[0]
```

Comment:
I think it's probably less surprising to do the `[0]` dereferencing at the point where `PARLIAMENTARY_SESSION` is defined, then you don't even need the comment :)